prow/config/org/org.go

```diff
@@ -22,6 +22,12 @@ import (
 	"k8s.io/test-infra/prow/github"
 )
 
+// FullConfig stores the full configuration to be used by the tool, mapping
```

Comment:
Why don't we a) move this out of prow/config and b) peribolos out of prow?

Reply:
Both are larger changes that can be follow-ups